## Error for 1_Default_LightGBM

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 2_Default_Xgboost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 3_Default_CatBoost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 4_Default_NeuralNetwork

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 5_Default_RandomForest

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 5_LightGBM

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 1_Xgboost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 9_CatBoost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 13_RandomForest

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 17_NeuralNetwork

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 6_LightGBM

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 2_Xgboost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 10_CatBoost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 14_RandomForest

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 18_NeuralNetwork

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 7_LightGBM

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 3_Xgboost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 11_CatBoost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 15_RandomForest

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 19_NeuralNetwork

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 8_LightGBM

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 4_Xgboost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 12_CatBoost

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 16_RandomForest

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

## Error for 20_NeuralNetwork

'TfidfVectorizer' object has no attribute 'get_feature_names_out'
Traceback (most recent call last):
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 1084, in _fit
    trained = self.train_model(params)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\base_automl.py", line 371, in train_model
    mf.train(results_path, model_subpath)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\model_framework.py", line 184, in train
    X_train, y_train, sample_weight = self.preprocessings[
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\preprocessing.py", line 165, in fit_and_transform
    t.fit(X_train, col)
  File "C:\Users\Anjani\Anaconda3\lib\site-packages\supervised\preprocessing\text_transformer.py", line 27, in fit
    for f in list(self._vectorizer.get_feature_names_out()):
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names_out'


Please set a GitHub issue with above error message at: https://github.com/mljar/mljar-supervised/issues/new

